services/jd_services.h

```diff
@@ -40,7 +40,7 @@ typedef struct {
 } servo_params_t;
 
 // Servo control service.
-void servo_init(const servo_params_t *params);
+void servo_init(const servo_params_t *params, uint8_t power_pin);
```

Comment:
why not include it in servo_params_t ?

Reply:
It seemed like `servo_params_t` can be updated by the client. Seemed like a bad idea to have the value potentially overridden... can update if this is not the case!

Reply:
what can be updated (what is exposed) depends on `REG_DEFINITION` - there's:

```c
    REG_U8(JD_REG_PADDING),       // pin
    REG_U8(JD_REG_PADDING),       // fixed
```

for the two fields of params that should not be exposed.

You can also put fields behind ro/const registers and the client won't be able to update it either (doesn't make sense for pin, just in general)